packages/psss-server/src/routes/LogoutRoute.ts

```diff
@@ -3,12 +3,20 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { Route } from './Route';
+import { UnauthenticatedRoute } from './UnauthenticatedRoute';
 
-export class LogoutRoute extends Route {
+export class LogoutRoute extends UnauthenticatedRoute {
   async buildResponse() {
-    await this.sessionModel.deleteById(this.sessionCookie.id);
-    this.sessionCookie?.reset();
+    const sessionId = this.sessionCookie?.id;
```

Comment:
I thought that it would be better if the logout route didn't return an error if it was called without a sessionCookie in the request (which means it had to be unauthenticated) as it could lead to a few issues including potentially getting stuck in a 401 loop.